package.json

```diff
@@ -21,12 +21,12 @@
   },
   "homepage": "https://github.com/seek-oss/browserslist-config-seek#readme",
   "devDependencies": {
-    "@commitlint/cli": "^6.0.5",
-    "browserslist": "3.0.0",
-    "commitlint-config-seek": "1.0.0",
+    "@commitlint/cli": "^6.1.2",
+    "browserslist": "3.1.1",
```

Comment:
Since we're only using browserslist to check that it's valid config, it probably makes sense to set this to `^3.1.1`.